.github/workflows/coverage-pr-base.yml

```
@@ -38,3 +35,8 @@ jobs:
             --slug PyO3/pyo3 \
             --token ${{ secrets.CODECOV_TOKEN }} \
             --service github
+        env:
+          # Don't put these in bash, because we don't want the expansion to
+          # risk code execution
+          BASE_REF: "refs/heads/{{ github.event.pull_request.base.ref }}"
```

```suggestion
          BASE_REF: "refs/heads/${{ github.event.pull_request.base.ref }}"
```
I think this one is missing a "$" for expansion? At least it is currently broken in #4768

---

Sigh, yes.

---

Oops! :see_no_evil: